Bluedot-Innovation/Bluedot-React-Native-Minimal-Integration
# Po/multiple events

to test:

1. Initialise the SDK for the first time after install
2. Start Geotriggering (with FG or with BG, either will repro this bug)
3. Trigger a zone (eg. a zone set up over your location). Observe a single notification on the device when the event triggers, as expected.
4. Reset SDK
5. Start Geotriggering again (with FG or with BG, either will repro this bug)
6. Trigger a zone (eg. a zone set up over your location).

expected: single notification
